curp/src/server/raw_curp/mod.rs

```diff
@@ -1030,10 +1030,12 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
     }
 
     /// Check if all followers have caught up with the leader
-    pub(super) fn is_synced(&self) -> bool {
+    pub(super) fn is_synced(&self, node_id: ServerId) -> bool {
```

Comment:
Please also update the comment of this function

Reply:
Updated